lib/Zonemaster/Engine/Test/Delegation.pm

```diff
@@ -130,24 +130,24 @@ sub translation {
         IPV6_DISABLED            => 'IPv6 is disabled, not sending "{rrtype}" query to {ns}/{address}.',
         IS_NOT_AUTHORITATIVE     => "Nameserver {ns} response is not authoritative on {proto} port 53.",
         NAMES_MATCH              => "All of the nameserver names are listed both at parent and child.",
-        NOT_ENOUGH_IPV4_NS_CHILD => "Child does not list enough ({count}) IPv4 nameserver addresses ({addrs}). "
+        NOT_ENOUGH_IPV4_NS_CHILD => "Child does not list enough ({count}) nameservers that resolve to IPv4 addresses ({addrs}). "
           . "Lower limit set to {minimum}.",
-        NOT_ENOUGH_IPV4_NS_DEL => "Delegation does not list enough ({count}) IPv4 nameserver addresses ({addrs}). "
+        NOT_ENOUGH_IPV4_NS_DEL => "Delegation does not list enough ({count}) nameservers that resolve to IPv4 addresses ({addrs}). "
           . "Lower limit set to {minimum}.",
-        NOT_ENOUGH_IPV6_NS_CHILD => "Child does not list enough ({count}) IPv6 nameserver addresses ({addrs}). "
+        NOT_ENOUGH_IPV6_NS_CHILD => "Child does not list enough ({count}) nameservers that resolve to IPv6 addresses ({addrs}). "
           . "Lower limit set to {minimum}.",
-        NOT_ENOUGH_IPV6_NS_DEL => "Delegation does not list enough ({count}) IPv6 nameserver addresses ({addrs}). "
+        NOT_ENOUGH_IPV6_NS_DEL => "Delegation does not list enough ({count}) nameservers that resolve to IPv6 addresses ({addrs}). "
           . "Lower limit set to {minimum}.",
         NOT_ENOUGH_NS_CHILD => "Child does not list enough ({count}) nameservers ({ns}). Lower limit set to {minimum}.",
         NOT_ENOUGH_NS_DEL   => "Parent does not list enough ({count}) nameservers ({glue}). "
           . "Lower limit set to {minimum}.",
-        NO_IPV4_NS_CHILD => "Child lists no ({count}) IPv4 nameserver addresses ({addrs}). If any were present, "
+        NO_IPV4_NS_CHILD => "Child lists no nameserver that resolves to IPv4 address. If any were present, "
```

Comment:
"resolves to IPv4 address" -> "resolves to an IPv4 address"

More of these below.